site/docs/configuration/beacon.md

```
@@ -0,0 +1,347 @@
+
+:::warning
+Heatsoaking is an important part of achieving consistent first (and early) layers, especially on larger printers. RatOS includes a new adaptive heatsoaking feature that uses beacon proximity measurements to determine when the printer has reached thermal stability. This feature is is enabled by default for V-Core 4.x printers. Fixed-duration heatsoaking is also available, but must be configured. See [Heatsoaking](heatsoaking.md) for more information.
+:::
+
+1. Run `BEACON_RATOS_CALIBRATE BED_TEMP=85 CHAMBER_TEMP=45`. Use your target temperature for the `BED_TEMP` and `CHAMBER_TEMP` parameter. `CHAMBER_TEMP` is optional, and can be omitted.
+
+The automated beacon calibration will run the following calibrations and tests, which can also be used individually. Please make sure to read every section before starting the calibration.
+
+- [Initial calibration](#1-initial-calibration)
+- [Beacon latency check](#2-beacon-latency-check)
+- [Temperature expansion calibration](#3-temperature-expansion-calibration) (for non IDEX printer)
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)